resources/todomvc/big-dom-generator/utils/buildComplex.js

```diff
@@ -0,0 +1,76 @@
+const fs = require("fs").promises;
```

Comment:
Should we use the sync version of all the tools?

Indeed in all build files, the result of `buildComplex` is never `await`ed. This works because node automatically waits for the completion of promises (I think), but we might have surprises in the future if we want to do more things in the build files.
What do you think?

Reply:
@julienw Yeah, that works. 

[`fs.copyFileSync`](https://nodejs.org/api/fs.html#fscopyfilesyncsrc-dest-mode) doesn't recursively copy directories so I would need to use [`fs.cpSync(src, dest[, options])`](https://nodejs.org/api/fs.html#fscpsyncsrc-dest-options) which is labeled as [Stability: 1](https://nodejs.org/api/documentation.html#stability-index) - Experimental.

There is also `fs-extra`. Which approach do you like, or do you have another suggestion?

Reply:
I think I'd use `cpSync`, it's present since node v16 and I doubt the behavior changes much in the future for this usage.

Reply:
Done.